__tests__/components/Footer/Footer.test.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { render, cleanup, fireEvent } from '@testing-library/react-native';
 import Footer from '../../../src/components/Footer/Footer';
-
+import Store from '../../../src/context/Store';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Store` is added but it's not clear why this is necessary as the old hunk didn't require it. If `Store` is a context provider, it should be used at a higher level in your component tree and not within individual components or tests. However, if you're using it to wrap your component for testing purposes, then it's fine.

```diff
+ import Store from '../../../src/context/Store';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->